sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/DiskSkuTypes.java

```diff
@@ -26,6 +26,9 @@ public final class DiskSkuTypes {
     /** Static value ULTRA_SSD_LRS for DiskSkuTypes. */
     public static final DiskSkuTypes ULTRA_SSD_LRS = new DiskSkuTypes(DiskStorageAccountTypes.ULTRA_SSD_LRS);
 
+    /** Static value PREMIUM_V2_LRS for DiskSkuTypes. */
+    public static final DiskSkuTypes PREMIUM_V2_LRS = new DiskSkuTypes(DiskStorageAccountTypes.PREMIUM_V2_LRS);
```

Comment:
Is there any other V2 ?

Reply:
Currently only PREMIUM SSD V2:
https://learn.microsoft.com/en-us/azure/virtual-machines/disks-types